tests/php/_inc/lib/test_class.rest-api-endpoints.php

```diff
@@ -711,6 +711,9 @@ public function test_setting_enum_save() {
 		$response = $this->create_and_get_request( 'settings', array( 'carousel_background_color' => 'black' ), 'POST' );
 		$this->assertResponseStatus( 200, $response );
 
+		// It should also save correctly with a multi-part POST body.
```

Comment:
Here and elsewhere, the term "multi-part" is incorrect and should be removed.  This is just a regular POST request with Content-Type `application/x-www-form-urlencoded`.  A multipart request has Content-Type `multipart/form-data` and looks very different, usually containing uploaded files.  Examples of each type of request:

- http://stackoverflow.com/questions/14551194/how-are-parameters-sent-in-an-http-post-request#14551320
- http://stackoverflow.com/questions/913626/what-should-a-multipart-http-request-with-multiple-files-look-like/913749#913749